less/site.less

```diff
@@ -221,7 +221,7 @@ hr {
 }
 
 img.scalable {
-    max-width: 100%;
+    width: 100%;
```

Comment:
seems risky 😨 this class is used 9 times in udata core alone

Reply:
Move Fast and Break Things

Reply:
poke @jdesboeufs :trollface: